master/buildbot/process/buildrequestdistributor.py

```diff
@@ -1231,20 +1235,22 @@ def _maybeResumeBuildsOnBuilder(self):
 
         brids = [br.id for br in breqs]
         yield self.master.db.buildrequests.updateBuildRequests(brids, results=BEGINNING)
+          
+        buildDefered = self.katanaBuildChooser.bldr.maybeResumeBuild(slave, buildnumber, breqs)
+        buildDefered.addErrback(self._resume)
```

Comment:
@sualitu I think we may need to pass down the  breqs, builderName parameters to _resume function